README.md

```
@@ -49,3 +50,6 @@ JFood é uma aplicação que permite o gerenciamento de usuários e um cardápio
    
 ## Postman Collection
 - [Collection](https://documenter.getpostman.com/view/2828428/2sB2cVg2vU)
+
+## Arquitetura
+![](imgs/JFood-architecture.PNG)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)